### ISSUE TITLE
Using "showPreviewContainer=false" uploadFile method is not called

### ISSUE THREAD

**Issue by EJSpawn** [vugar005/ngx-awesome-uploader#11]
I used the DemoFilePickerAdapter to start, and worked fine, i add a field observable to catch the upload % done, and worked too, when i disabled the preview container  the uploadFile method stop be called!

component.html
```
<ngx-file-picker
    [adapter]="adapter"  
    [enableCropper]="true" 
    [showPreviewContainer]="false">
</ngx-file-picker>
```
adapter
```
export class FsExplorerFilePickerAdapter extends FilePickerAdapter {
    private _loaded = new BehaviorSubject<number>(0);
    public loaded = this._loaded.asObservable();

    constructor(private http: HttpClient) {
        super();
    }
    public uploadFile(fileItem: FilePreviewModel) {
        console.log('test');
        const form = new FormData();
        form.append('file', fileItem.file);
        const api = 'https://demo-file-uploader.free.beeceptor.com';
        const req = new HttpRequest('POST', api, form, { reportProgress: true });
        return this.http.request(req)
            .pipe(
                map((res: HttpEvent<any>) => {
                    if (res.type === HttpEventType.Response) {
                        return res.body.id.toString();
                    } else if (res.type === HttpEventType.UploadProgress) {
                        // Compute and show the % done:
                        const UploadProgress = +Math.round((100 * res.loaded) / res.total);
                        this._loaded.next(UploadProgress);
                        return UploadProgress;
                    }
                })
            );
    }
    public removeFile(fileItem: FilePreviewModel): Observable<any> {
        console.log(fileItem.fileId);
        const removeApi = 'https://file-remove-demo.free.beeceptor.com';
        return this.http.post(removeApi, { id: fileItem.fileId });
    }
}
```

component.ts
```
import { Component, OnInit } from '@angular/core';
import { FsExplorerFilePickerAdapter } from 'src/app/core/adapters/fs-explorer-filepicker.adapter';
import { HttpClient } from '@angular/common/http';

@Component({
  selector: 'app-fs-explorer',
  templateUrl: './fs-explorer.component.html',
  styleUrls: ['./fs-explorer.component.scss']
})
export class FsExplorerComponent implements OnInit {
  public adapter = new FsExplorerFilePickerAdapter(this.http);
  constructor(private http: HttpClient) { }

  ngOnInit() {
    this.adapter.loaded.subscribe(val => console.log(val));
  }

}
```

**Comment by vugar005** [comment]
Hi @EJSpawn . Thanks for feedback. It is expected since uploadFile method is triggered inside preview container .

**Comment by vugar005** [comment]
The issue is fixed from version 9.0.1
